src/SFA.DAS.AssessorService.Data.IntegrationTests/Services/TestWebConfiguration.cs

```diff
@@ -11,8 +11,6 @@ public class TestWebConfiguration : IWebConfiguration
         public ClientApiAuthentication AssessorApiAuthentication { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public NotificationsApiClientConfiguration NotificationsApiClientConfiguration { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public string AssessmentOrgsApiClientBaseUrl { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
This was possibly the FAT v1 connection - which has been replaced by the Outer API

Reply:
this has been removed in a separate branch by Dan A as part of their teams decom work

https://github.com/SkillsFundingAgency/das-assessor-service/pull/1195#pullrequestreview-665502908